src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -88,26 +106,110 @@ class DocumentStatisticsAnnotationData {
     // ----------------------------------------------------------------------
     using FrequencyMap                      = std::unordered_map<std::string, std::uint32_t>;
```

Comment:
Are these 2 using statements necessary after introducing FrequencyAndIndexMap?

Reply:
Looks there are not necessary in the AnnotationData class, removed.